api/machine/machine.proto

```diff
@@ -258,6 +258,7 @@ message LogsRequest {
   // driver might be default "containerd" or "cri"
   common.ContainerDriver driver = 3;
   bool follow = 4;
+  int32 tailLines = 5;
```

Comment:
nit: the convention in protobuf is to use snake case.

Reply:
thanks, fixed!